### DISCUSSION TITLE
SBOM GENERATION

### DISCUSSION THREAD

**Discussion by Vishnu-2810** [aquasecurity/trivy#5927] [Q&A] [answered]
### Question

Is it possible to get the list of softwares alone without analysing the security vulnerability of those components?
If possible ,let me know how to generate it
Thanks in advance @DmitriyLewen  

### Target

None

### Scanner

None

### Output Format

None

### Mode

None

**Comment by DmitriyLewen** [comment, marked as answer]
Hello @Vishnu-2810 

You can use one of [SBOM](https://aquasecurity.github.io/trivy/v0.48/docs/supply-chain/sbom/) formats for this.

Regards ,Dmitriy
